css/components/badge.css

```diff
@@ -53,4 +53,17 @@
 
 .badge-offline {
     background-color: var(--light-gray);
-}
\ No newline at end of file
+}
+
+/* size varient */
+.icon-lg{
+    font-size: 3rem;
+}
+
+.icon-m{
+    font-size: 2rem;
+}
+
+.icon-s{
+    font-size: 1rem;
+}
```

Comment:
all variable looks good to me!

Reply:
you can use sm instead of s for better readability.

Reply:
made the required changes please look into it and approve the changes.